components/ActionBar/index.js

```diff
@@ -0,0 +1,146 @@
+import React, { Component, Fragment } from 'react'
+import PropTypes from 'prop-types'
+import { css } from 'glamor'
+
+import IconLink from '../IconLink'
+import ShareOverlay from './ShareOverlay'
+import withT from '../../lib/withT'
+
+import { colors } from '@project-r/styleguide'
+
+const styles = {
+  buttonGroup: css({
+    '@media print': {
+      display: 'none'
+    }
+  })
+}
+
+class ActionBar extends Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      showShareOverlay: false
+    }
+
+    this.toggleShare = () => {
+      this.setState({
+        showShareOverlay: !this.state.showShareOverlay
+      })
+    }
+  }
+  render () {
+    const {
+      t,
+      url,
+      tweet,
+      emailSubject,
+      emailBody,
+      emailAttachUrl,
+      download,
+      dossierUrl,
+      fill,
+      onAudioClick,
+      onPdfClick,
+      pdfUrl,
+      shareOverlayTitle
+    } = this.props
+    const { showShareOverlay } = this.state
+
+    const icons = [
+      {
+        icon: 'share',
+        href: url,
+        onClick: e => {
+          if (e.currentTarget.nodeName === 'A' &&
```

Comment:
Rm this code. Trying to open the share overlay in a new window is not supported. It should `e.preventDefault()` and show the overlay in the current window.